e2e/tests/helpers.ts

```
@@ -15,29 +25,97 @@ export async function dismissConsoleTour(page: Page): Promise<void> {
+    console.warn('Modal box not visible after backdrop appeared');
+    return;
+  }
+
+  const dismissStrategies = [
+    // PatternFly v6 selectors
+    { name: 'Skip tour (exact)', locator: modalBox.locator('button:text-is("Skip tour")') },
+    { name: 'Skip (exact)', locator: modalBox.locator('button:text-is("Skip")') },
+    { name: 'Get started (exact)', locator: modalBox.locator('button:text-is("Get started")') },
+    { name: 'Close button (aria-label)', locator: modalBox.locator('button[aria-label="Close"]') },
+    { name: 'Close button (title)', locator: modalBox.locator('button[title="Close"]') },
+    { name: 'Times icon button', locator: modalBox.locator('button .pf-v6-c-button__icon .fa-times, button .pf-c-button__icon .fa-times') },
```

ESLint

[error] Replace `·name:·'Times·icon·button',·locator:·modalBox.locator('button·.pf-v6-c-button__icon·.fa-times,·button·.pf-c-button__icon·.fa-times')` with `⏎······name:·'Times·icon·button',⏎······locator:·modalBox.locator(⏎········'button·.pf-v6-c-button__icon·.fa-times,·button·.pf-c-button__icon·.fa-times',⏎······),⏎···`

(prettier/prettier)